bluemira/codes/process/_teardown.py

```diff
@@ -140,220 +161,109 @@ def _load_mfile(self, path: str, recv_all: bool):
-            ("r_fw_ob_in", "Outboard first wall inner radius", r_fw_ob_in),
-            ("r_vv_ob_in", "Outboard vessel inner radius", r_vv_ob_in),
-        ]
-        for radial_param in radial_params:
-            rb.add_parameter(*radial_param, "m", None, PROCESS_NAME)
-        self.params["Radial Build"] = rb
-
-    def extract_outputs(self, outputs: Union[List, str]) -> List[float]:
-        """
-        Searches MFile for variable.
-
-        Outputs defined in bluemira variable names if they are mapped
-        otherwise process variable names are the default
-
-        Parameters
-        ----------
-        outputs: Union[List, str]
-            parameter names to access
-
-        Returns
-        -------
-        List of values
+        rtfin = data["bore"] + data["ohcth"] + data["precomp"] + data["gapoh"]
+        r_ts_ib_in = rtfin + data["tfcth"] + data["tftsgap"] + data["thshield"]
+        r_vv_ib_in = r_ts_ib_in + data["gapds"] + data["d_vv_in"] + data["shldith"]
+        r_fw_ib_in = r_vv_ib_in + data["vvblgap"] + data["blnkith"] + data["fwith"]
+        r_fw_ob_in = r_fw_ib_in + data["scrapli"] + 2 * data["rminor"] + data["scraplo"]
+        r_vv_ob_in = r_fw_ob_in + data["fwoth"] + data["blnkoth"] + data["vvblgap"]
+        return {
+            "rtfin": rtfin,
+            "r_ts_ib_in": r_ts_ib_in,
+            "r_vv_ib_in": r_vv_ib_in,
+            "r_fw_ib_in": r_fw_ib_in,
+            "r_fw_ob_in": r_fw_ob_in,
+            "r_vv_ob_in": r_vv_ob_in,
+        }
+
+
+def update_mappings(old_mappings: Dict[str, ParameterMapping]):
+    """
+    Convert old PROCESS mappings to new ones
 
-        """
-        out: List[float] = []
-        if isinstance(outputs, str):
-            # Handle single variable request
-            outputs = [outputs]
-        for var in outputs:
-            if isinstance(var, list):
-                for v in var:
-                    if found := self._find_var_in_frame(v, out):
-                        break
-            else:
-                found = self._find_var_in_frame(var, out)
-
-            if not found:
-                process_var = self.btop_mapping.get(var, var)
-                bluemira_var = "N/A" if var == process_var else var
-                out.append(0.0)
-                bluemira_warn(
-                    f"bluemira variable '{bluemira_var}' a.k.a. "
-                    f"PROCESS variable '{process_var}' "
-                    "not found in PROCESS output. Value set to 0.0."
-                )
-        return out
+    Parameters
+    ----------
+    old_mappings: dict
+        dictionary of parameter mappings
 
-    def _find_var_in_frame(self, var, out):
-        """
-        Find variable value in parameter frame
+    Returns
+    -------
+    new_mappings: dict
+        dictionary of new parameter mappings
+    """
+    new_mappings = {}
+    for key, val in old_mappings.items():
+        new_name = update_obsolete_vars(val.name)
```

Comment:
Hmm, this doesn't look like something we previously dealt with. Also, I can't work out how `update_obsolete_vars` would return a list, as PROCESS's `OBS_VARS` dict's values only contain strings (in my version anyway)

Reply:
In later versions of PROCESS 1 variable has been split into two so the OBS_VARS returns a list of two string instead of one...which is difficult lets say